packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -278,6 +279,11 @@ export function BlockSettingsDropdown( {
 										</MenuItem>
 									</>
 								) }
+
+								<__unstableCommentIconFill.Slot
+									fillProps={ { onClose } }
+								/>
```

Comment:
Do we really need another slot, we already have `BlockSettingsMenuControls` or `__unstableBlockSettingsMenuFirstItem` in this menu already. Can we use one of these?

Reply:
Based on [feedback](https://github.com/WordPress/gutenberg/pull/60622#issuecomment-2309620757), we've added a new slot to display the comment icon below the "Add after" option. If we use the `__unstableBlockSettingsMenuFirstItem` slot, the comment icon will instead appear at the [top of the menu group](https://prnt.sc/aoLn8OCRrD2p).

Reply:
I know, how important the position is. Can it be at the top or at the end of the menu instead ?

Reply:
Yes, we can position it at the top, as shown [here](https://prnt.sc/aoLn8OCRrD2p). We can also confirm with [jasmussen](https://github.com/jasmussen) to ensure he's okay with this change.

Reply:
@jasmussen @jameskoster